CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 platformdirs Changelog
 ======================
 
+platformdirs 2.4.2
+------------------
+- Fix OSError when platformdirs is runned from subsystem in Termux
```

Comment:
I'd position this not as a bug but as a feature.:`Add support for Termux subsystems`. So let's make it `2.5.0`.